.github/pull_request_template.md

```diff
@@ -1,9 +1,15 @@
-Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):
-
 Resolves https://jira.corp.adobe.com/browse/MWPW-NUMBER
-
 QA Checklist: https://wiki.corp.adobe.com/display/adobedotcom/M@S+Engineering+QA+Use+Cases
 
+Please do the steps below before submitting your PR for a code review or QA
+
+[x] C1. Cover code with Unit Tests
+[x] C2. Add a Nala test (double check with #fishbags if nala test is needed)
+[x] C3. Verify all Checks are green (unit tests, nala tests, psi)
+[x] C4. PR description contains working Test Page link where the feature can be tested
+[x] C5: you are ready to do a demo from Test Page in PR (bonus: write a working demo script that you'll use on Thursday, you can eventually put in your PR)
+[x] C.6 read your Jira one more time to validate that you've addressed all AC's and nothing is missing
```

Comment:
- [ ] C1. Cover code with Unit Tests
- [ ] C2. Add a Nala test (double check with #fishbags if nala test is needed)
- [ ] C3. Verify all Checks are green (unit tests, nala tests, psi)
- [ ] C4. PR description contains working Test Page link where the feature can be tested
- [ ] C5: you are ready to do a demo from Test Page in PR (bonus: write a working demo script that you'll use on Thursday, you can eventually put in your PR)
- [ ] C.6 read your Jira one more time to validate that you've addressed all AC's and nothing is missing

Reply:
I agree checkboxes must be unchecked.